plugins/aws/ec2/openFTP.spec.js

```diff
@@ -0,0 +1,139 @@
+var expect = require('chai').expect;
+const openFTP = require('./openFTP');
+
+const describeSecurityGroups = [
+    {
+        "Description": "default VPC security group",
+        "GroupName": "default",
+        "IpPermissions": [],
+        "OwnerId": "111122223333",
+        "GroupId": "sg-aa941691",
+        "IpPermissionsEgress": [],
+        "VpcId": "vpc-99de2fe4"
+    },
+    {
+        "Description": "Master group for Elastic MapReduce created on 2020-08-31T17:07:19.819Z",
+        "GroupName": "ElasticMapReduce-master",
+        "IpPermissions": [
+            {
+                "FromPort": 0,
+                "IpProtocol": "tcp",
+                "IpRanges": [
+                    {
+                        "CidrIp": "0.0.0.0/0"
+                    }
+                ],
+                "Ipv6Ranges": [],
+                "PrefixListIds": [],
+                "ToPort": 65535,
```

Comment:
same here FTP port is 20 and 21. But in failing case we dont have test cases with these two open

Reply:
I am sending all ports open which are 0-65535 so technically that covers 'tcp': [20, 21] as well.